clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/securitygroup/SecurityGroupLookupFactory.groovy

```diff
@@ -249,6 +250,14 @@ class SecurityGroupLookupFactory {
       this.amazonEC2 = amazonEC2
     }
 
+    void updateIngress(List<IpPermission> ipPermissionsToUpdate) {
+      amazonEC2.updateSecurityGroupRuleDescriptionsIngress(new UpdateSecurityGroupRuleDescriptionsIngressRequest(
+        groupId: securityGroup.groupId,
+        ipPermissions: ipPermissionsToUpdate
+      ))
+      securityGroup.ipPermissions.addAll(ipPermissionsToUpdate)
```

Comment:
is this right? since it's just a list, won't this mean that an updated iprule will show up twice? once with old description and once with new?
I am not sure it's used today, but even if not, in the future someone might assume it works differently

Reply:
I have removed it , not sure if its used today